server/src/test/scala/scenarios/ArrivalsSimlationSpec.scala

```diff
@@ -0,0 +1,147 @@
+      ApiPaxTypeAndQueueCount(NonVisaNational, Queues.NonEeaDesk, 5.0, None),
+      ApiPaxTypeAndQueueCount(VisaNational, Queues.NonEeaDesk, 6.0, None),
+      ApiPaxTypeAndQueueCount(Transit, Queues.Transfer, 7.0, None),
+      ApiPaxTypeAndQueueCount(EeaMachineReadable, Queues.EGate, 8.0, None)
+    ), ApiSplitsWithHistoricalEGateAndFTPercentages, Option(EventTypes.DC))
+    val expected: ApiFlightWithSplits = ApiFlightWithSplits(flight, Set(splits))
+
+    val csvFromFlightWithSplits = TerminalFlightsWithActualApiSummary(
+      Seq(expected),
+      Exports.millisToUtcIsoDateOnly,
+      Exports.millisToUtcHoursAndMinutes,
+      PcpPax.bestPaxEstimateWithApi
+    ).toCsvWithHeader
+
+    val result: ApiFlightWithSplits = ArrivalImporter(csvFromFlightWithSplits, terminal).head
+
+    result === expected
+  }
+
+  "Given an arrival CSV row then I should get back a representative ApiPaxTypeAndQueueCount split for the arrival" >> {
+
+    val csvLines = ArrivalImporter.toLines(csv)
+    val headers = ArrivalImporter.csvHeadings(csvLines)
+
+    val result = ArrivalImporter.lineToSplits(ArrivalImporter.lineToFields(csvLines(1)), headers)
+
+    val expected: Set[Splits] = Set(Splits(
+      Set(
+        ApiPaxTypeAndQueueCount(B5JPlusNational, Queues.EeaDesk, 0, None),
+        ApiPaxTypeAndQueueCount(B5JPlusNational, Queues.EGate, 0, None),
+        ApiPaxTypeAndQueueCount(EeaMachineReadable, Queues.EeaDesk, 8.0, None),
+        ApiPaxTypeAndQueueCount(EeaNonMachineReadable, Queues.EeaDesk, 7.0, None),
+        ApiPaxTypeAndQueueCount(NonVisaNational, Queues.NonEeaDesk, 1.0, None),
+        ApiPaxTypeAndQueueCount(VisaNational, Queues.NonEeaDesk, 1.0, None),
+        ApiPaxTypeAndQueueCount(Transit, Queues.Transfer, 13.0, None),
+        ApiPaxTypeAndQueueCount(EeaMachineReadable, Queues.EGate, 46.0, None)
+      ),
+      ApiSplitsWithHistoricalEGateAndFTPercentages, Option(EventTypes.DC)
+    ))
+
+    result === expected
+  }
+
+  "Given a csv of arrivals for a day then I should get a desks and queues export for that day" >> {
+
+    val flightsWithSplits = ArrivalImporter(csv, terminal)
+
+    val lhrHalved = Lhr.config.copy(
+      minMaxDesksByTerminalQueue24Hrs = Lhr.config.minMaxDesksByTerminalQueue24Hrs.mapValues(_.map {
+        case (q, (_, max)) =>
+          val openDesks = max.map(x => x / 2)
+          q -> (openDesks, openDesks)
+      }),
+      eGateBankSize = 5,
+      slaByQueue = Lhr.config.slaByQueue.mapValues(_ => 15)
+    )
+    val fws = FlightsWithSplits(flightsWithSplits.map(f => f.unique -> f).toMap)
+
+    val portStateActor = system.actorOf(Props(new ArrivalCrunchSimulationActor(fws)))
+    val dawp = DesksAndWaitsPortProvider(lhrHalved, Optimiser.crunch, PcpPax.bestPaxEstimateWithApi)
+
+    val (runnableDeskRecs, _): (SourceQueueWithComplete[MillisSinceEpoch], UniqueKillSwitch) = RunnableDeskRecs(portStateActor, dawp, PortDeskLimits.fixed(lhrHalved)).run()
```

Comment:
We should improve the RunnableDeskRecs service to take terminal specific requests for crunching as we crunch all terminals now even if only one has any changes